docs/src/main/sphinx/connector/hive.rst

```diff
@@ -388,6 +388,12 @@ Property Name                                      Description
 ``hive.query-partition-filter-required``           Set to ``true`` to force a query to use a partition filter.   ``false``
                                                    You can use the ``query_partition_filter_required`` catalog
                                                    session property for temporary, catalog specific use.
+
+``hive.statistics_enabled``                        Enables :doc:`/optimizer/statistics`. The equivalent          ``true``
+                                                   :doc:`catalog session property </sql/set-session>` 
+                                                   is ``statistics_enabled`` for session specific use. 
+                                                   Set to ``false`` to disable statistics, this also disables
+                                                   :doc:`/optimizer/cost-based-optimizations`.
```

Comment:
nitpicking - this is technically true but a side effect of the fact that stats become missing. i.e. there might be future cost-based optimisations which might not be dependent on stats (or we could generate some rough stats during runtime) and then this statement would be false.

Reply:
Thanks @hashhar, thats good info to have. 

@mosabua and @m57lyra - should I future proof for the above use cases or assume that we'll update the definition as needed?

Reply:
I could do a minor edit to say this or something similar:

> Set to ``false`` to disable statistics. Disabling statistics
   means that :doc:`/optimizer/cost-based-optimizations` can 
   not make smart decisions about the query plan.

Reply:
+1 on this minor edit.

Reply:
Implemented edit.